rust/common/error/macros.rs

```diff
@@ -108,4 +131,65 @@ macro_rules! error_messages {
             return format!($body, _0, _1, _2, _3)
         }
     };
+
+    (@payload $self:ident, $format:ident, $error_name:ident) => {
```

Comment:
It'd make more sense for `$error_name` to come before `$format` here, as it belongs with `$self`.